docs/projects/t1-2022/thoth-tech-leadership/t2-leadership-project-recommendations.md

```diff
@@ -0,0 +1,113 @@
+# T1 Leadership Project Recommendation for T2
+
+## General recommendations
+
+- Clear messaging early on
+- Clear messaging from the beginning around general delivery schedule + self driven expectations
+- Weekly written team progress reports to Area Leads
+- Partnering experienced seniors with juniors where possible
+- Balance of High achieving students to drive the delivery
+- Product Lead + Area lead merged
+- Addition of more area leads
+- QA team
+- Documentation team part of Internal Systems
+- Delivery lead training in week 0
+
+Implement recommendations from T1 Delivery leads
+
+## Onboarding form improvements:
+
+- skill confidence (1-5) eg. programming, troubleshooting, documentation, project collaboration etc
+- do you want to work towards leadership?
+- Grade aims in onboarding form for team splits
+
+## OnTrack
+
+### Front-end Migrations
+
+- Upskilling links - migration guide
+- Front end experience, JavaScript/TypeScript
+- Utilising seniors early on
+- Partnering experienced seniors with juniors
+- Smaller meetings early on
+- Possibly two teams
+- 1 lead per 3/4 teams of 2
+
+### Deployment
+
+- Similar number of people - pipeline maybe 1 person
+- Make Google Cloud + Pipeline 1 sub group
+- Mat & Lachlan to take over google cloud work
+- Enhance Authentication: Balance of ComSci student + Cyber
+
+### Jupyter Notebook
+
+- Skills: good understanding of the programming language python so they can learn ruby-on-rails
+  quickly. The work that I would expect them to do has many similarities to that of python and i
+  feel that if they understand python they will have no difficulty understanding ruby.
+- Size: 2-3 students
+- Other options available if the next trimester team finishes early or if they get to many people
+  such as other conversion types. This is the same as how this trimester the Jupyter Notebook
+  Support team not only did the Jupyter Notebook to PDF conversion but also Microsoft Word to PDF.
+
+### Voice Verification
+
+- 5-6 is more than enough including delivery lead
+- Documentation focus + writing documentation
+- Higher balance of seniors to juniors
+- High confidence in programming
+- Balance of High achieving students to drive the delivery
+
+### Documentation
+
+- Move toward a Document Control role, having ownership over documentation and product documents.
+- Size: 6-10 students, sub teams for each product
+- Skills: good understanding of English language, understanding of markdown, strong engagement,
+  strong communication, and experience with GitHub
+- Good for people wanting to avoid coding, but still technical understanding to approve work.
+
+## SplashKit
```

Comment:
Just missing Applications
Operations is complete as of this tri (new epic created)